java/src/main/java/org/rocksdb/RawSstFileReaderIterator.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * <p>An iterator that yields a sequence of internal key/value pairs from a source.
```

Comment:
Is this Javadoc correct and specific to this iterator type?

Reply:
yeah